easybuild/easyconfigs/g/GAMESS-US/GAMESS-US-2013.5.1-ictce-5.4.0.eb

```diff
@@ -0,0 +1,20 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+##
```

Comment:
Is that the case even when a custom EasyBlock, such as (in this case) EB_GAMESS_minus_US, is to be used?